tools/mmp/driver.cs

```diff
@@ -1700,6 +1700,8 @@ static void CopyI18nAssemblies (I18nAssemblies i18n)
 
 				// The linker later gets angry if you copy in a read only assembly
 				CopyFileAndRemoveReadOnly (asm, Path.Combine (mmp_dir, filename));
+				foreach (var assembly in BuildTarget.Assemblies)
```

Comment:
You're iterating over all the assemblies inside a foreach of all the assemblies. Shouldn't this new foreach be placed outside the containing foreach?